tests/integrations/test_postgre_integration.py

```diff
@@ -0,0 +1,37 @@
+
+import psycopg2
```

Comment:
also I didn't see any postgresql dependency for development environment

Reply:
added dependencies to Pipfile